app/views/sufia/homepage/_home_header.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class="home_call_action col-xs-12 col-sm-4 pull-right">
-  <% if can? :view_share_work, GenericWork %>
+  <% if Sufia.config.always_display_share_button || can_ever_create_works? %>
```

Comment:
The idea here is that sufia by default has a "Share your work" button on the homepage. Some institiutions (Oregon Digital) don't want this turned on unless the user actually has permission. `can_ever_create_works?` is from CurationConcerns (originally from Curate I think).